tempodb/backend/s3/s3.go

```diff
@@ -26,8 +26,8 @@ import (
 )
 
 const (
-	s3KeyDoesNotExist  = "The specified key does not exist."
-	uptoHedgedRequests = 2
+	s3KeyDoesNotExistCode = "NoSuchKey"
```

Comment:
instead of redefining this can we reference the exported error code in the s3 package?

https://github.com/aws/aws-sdk-go/blob/main/service/s3/errors.go#L40

Reply:
@joe-elliott hi, changed